### PR TITLE
Remove commit_sha

### PR DESCRIPTION
**Pull request by JosephGoulden**

### CODE REVIEW

File: nix/pkgs/kld.nix [hunk 1]
```diff
@@ -51,7 +52,6 @@ craneLib.buildPackage {
       nativeBuildInputs = nativeBuildInputs ++ [ bitcoind cockroachdb ];
     };
     inherit cargoArtifacts;
-    COMMIT_SHA = lib.mkIf (self ? rev) self.rev;
```
Comment (Mic92) [inline nix/pkgs/kld.nix:54]: This cause a rebuild on every git push even if no code has changed. I rather would not have this as it increases the build times a lot.

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors cancel

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors cancel

**Comment by Mic92** [comment]
bors merge

**Comment by JosephGoulden** [comment]
bors merge

**Pull request merged**